mco_ovs_supervisor.yml

```diff
@@ -10,8 +10,6 @@ spec:
       version: 3.1.0
     systemd:
       units:
-        - enabled: true
```

Comment:
Awesome! That probably means we can remove the external-ovs labels too.

Reply:
Yep, I will do it in the next PR with the fix of the "setup-ovs-with-labels"